api/app/controllers/api/v1/instructors_controller.rb

```diff
@@ -54,10 +68,18 @@ def instructors_by_position
             end
         end
 
-        def validate_position_ids
-            params.require(:position_ids)
-            params[:position_ids].each do |id|
-                Position.find(id)
+        def instructor_create(instructor, filter = false)
+            if filter
+                result = error = instructors_by_position
+            else
+                result = instructor
+                error = {}
+            end
+            if instructor.save # passes Instructor model validation
+                render_success(result)
+            else
+                instructor.destroy!
+                render_error(instructor.errors, error)
```

Comment:
Why not `render_error(instructor.errors.full_messages.join("; "))`?

Reply:
Good catch. I'll fix that now. Thanks.